src/theme.ts

```diff
@@ -5,4 +5,8 @@ export const theme = extendTheme({
     semibold: 500,
     bold: 500,
   },
+  config: {
+    initialColorMode: "light",
+    useSystemColorMode: false,
```

Comment:
Why not using system color mode?

Reply:
We will consider giving the option to users in the future :)